staging/src/k8s.io/component-helpers/resource/helpers.go

```diff
@@ -144,22 +144,21 @@ func AggregateContainerRequests(pod *v1.Pod, opts PodResourcesOptions) v1.Resour
 	reqs := reuseOrClearResourceList(opts.Reuse)
 	var containerStatuses map[string]*v1.ContainerStatus
 	if opts.UseStatusResources {
-		containerStatuses = make(map[string]*v1.ContainerStatus, len(pod.Status.ContainerStatuses))
+		containerStatuses = make(map[string]*v1.ContainerStatus, len(pod.Status.ContainerStatuses)+len(pod.Status.InitContainerStatuses))
```

Comment:
kubectl has a similar method that needs to be updated: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/util/resource/resource.go#L40

Reply:
Updated `podRequests` method of kubectl, but it seems we don't need to update `podLimits`?